src/emailpro/domain/emailpro-domain.html

```diff
@@ -128,11 +128,11 @@
                                     <button class="oui-status oui-status_error" type="button"
                                             data-translate="emailpro_tab_domain_diagnostic_srv"
                                             data-ng-click="setAction('emailpro/domain/srv-autoconfig/emailpro-domain-srv-autoconfig', domain)"
-                                            data-ng-if="domain.srvValid === false">
+                                            data-ng-if="domain.srvValid === false && !exchange.isMXPlan">
```

Comment:
I know you just added `&& !exchange.isMXPlan`, but if you can, why not remove explicit boolean falsy test ? 
```suggestion
                                            data-ng-if="!domain.srvValid && !exchange.isMXPlan">
```